src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/Vilkår/BarnetsAlder/BarnetsAlderContext.ts

```diff
@@ -105,8 +110,47 @@ export const useBarnetsAlder = (
                 erEksplisittAvslagPåSøknad: erEksplisittAvslagPåSøknad.verdi,
             },
         }),
+        adopsjonsdato: useFelt<Date | undefined>({
+            verdi: undefined,
+            valideringsfunksjon: felt => {
+                if (!felt.verdi || !isValid(felt.verdi)) {
+                    return feil(felt, 'Adopsjonsdato må fylles ut når adopsjon er valgt');
+                } else {
+                    return ok(felt);
+                }
+            },
+            avhengigheter: { utdypendeVilkårsvurdering: utdypendeVilkårsvurdering.verdi },
+            skalFeltetVises: (avhengigheter: Avhengigheter) =>
+                avhengigheter?.utdypendeVilkårsvurdering.includes(
+                    UtdypendeVilkårsvurderingGenerell.ADOPSJON
+                ),
+        }),
     };
 
+    const [forrigeAdopsjonsdato, settForrigeAdopsjonsdato] = useState<IsoDatoString | undefined>();
+
+    const settAdopsjonsdatoTilDefault = () => {
+        felter.adopsjonsdato.validerOgSettFelt(
+            person.adopsjonsdato ? new Date(person.adopsjonsdato) : undefined
+        );
+    };
+
+    if (person.adopsjonsdato !== forrigeAdopsjonsdato) {
+        settForrigeAdopsjonsdato(person.adopsjonsdato);
+        settAdopsjonsdatoTilDefault();
+    }
```

Comment:
Dette sørger for å oppdatere adopsjonsdato-feltet hvis verdi fra backend endrer seg

Reply:
Kunne man satt`verdi` i adopsjonsdato-feltet direkte med `person.adopsjonsdato ? new Date(person.adopsjonsdato) : undefined`?  Er det et scenario dette ikke vil bli riktig?